ddtrace/appsec/utils.py

```diff
@@ -141,3 +143,38 @@ def parse_form_params(body):
             else:
                 req_body[key] = [prev_value, val]
     return req_body
+
+
+def parse_form_multipart(body):
+    # type: (unicode) -> dict[unicode, Any]
+    """Return a dict of form data after HTTP form parsing"""
+    import email
+    import json
+
+    import xmltodict
```

Comment:
nit: how about moving these imports to their corresponding `if`s?

Reply:
Those modules are probably already imported at the time we reach that piece of code anyway. I was wondering if I should put them at the start of the file.
I don't know, each possibility has different advantages or drawbacks.